Pocket/Pocket/Models/QueryResponse.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  QueryResponse.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 6/5/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public class QueryResponse {
+    public var query = Query()
+    public var result: Dictionary<AnyHashable, Any>?
+    public var decoder: Dictionary<AnyHashable, Any>?
+    public var decoded = false
+    public var error = false
+    public var error_msg = ""
+    
+    init(json: JSON) {
+        guard let responseObject = json.dictionaryObject else {
+            return
+        }
+        
+        if responseObject["query"] != nil{
+            query = Query.init()
+            query.query = responseObject["query"] as! Dictionary<AnyHashable, Any>
```

Comment:
Wouldn't this be better as:

`query = Query.fromDictionary(responseObject["query"] as! Dictionary<AnyHashable, Any>)`